client/tasks/tasks.tsx

```diff
@@ -70,6 +70,11 @@ export const Tasks: React.FC< TasksProps > = ( { query } ) => {
 		hideTaskList();
 	};
 
+	useEffect( () => {
+		document.body.classList.add( 'woocommerce-onboarding' );
+		document.body.classList.add( 'woocommerce-task-dashboard__body' );
```

Comment:
I know this is just migrating old code, but it seems like this never gets removed from the page when navigating away.  Not entirely sure why we need this body classes at all actually.

Okay to handle in a follow-up though- would you mind creating an issue if we don't remove here?

Reply:
Yeah @joshuatf , I actually toyed around with removing these earlier, but some of the styles for the various tasks depend on these body classes. I actually mentioned this in the notes above ^. I'm good with creating an issue for a follow-up 👍🏻

Reply:
[Done](https://github.com/woocommerce/woocommerce-admin/issues/7693)